web-client/src/pages/error-404.js

```diff
@@ -1,8 +1,30 @@
 import React from "react";
 import "../App.css";
+import { makeStyles } from "@material-ui/core/styles";
+import { Button } from "@material-ui/core/";
+import { NavLink } from "react-router-dom";
+import background from "../images/404.png";
 
-function Error404(props) {
-  return <h1>Error 404</h1>;
-}
+export default function Error404() {
+  const style = makeStyles({
+    photo: {
+      width: "90%",
+    },
+    linkButton: {
+      textDecoration: "none",
+      textTransform: "none",
+    },
+  })();
 
-export default Error404;
+  return (
+    <div>
+      <h1 className={style.title}>Oops! Page not found</h1>
+      <NavLink exact to="/" className={style.linkButton}>
+        <Button color="primary" variant="contained" className={style.linkButton}>
+          Go to home
+        </Button>
+      </NavLink>
+      <div><img className={style.photo} src={background} alt="404"/></div>
```

Comment:
```suggestion
      <div><img className={style.photo} src={background} alt=""/></div>
```

Reply:
must be alt after all

Reply:
yes but it's better to leave alt without text if this photo is only decorative